en/docs/deploy/aws.md

```diff
@@ -0,0 +1,88 @@
+# Deploying WSO2 Identity Server on Kubernetes using Minikube
+
+!!! info "Prerequisites"
```

Comment:
are there any concepts that we should be linking to in order to understand a bit more about minikube or kubernetes?

Reply:
We have linked to the official platform sites for the instructions but not linked off to concepts because the initial idea was that people who want to deploy the app in a certain platform, would choose it since they already are working on it or are familiar with it, and linking them off would only be distracting. WDYT?

Reply:
yeah, in that case it makes sense